src/Language/Haskell/LSP/Control.hs

```diff
@@ -67,12 +67,12 @@ runWithHandles hin hout dp h o captureFp = do
   hSetBuffering hout NoBuffering
   hSetEncoding  hout utf8
 
-  -- Delete existing recordings if they exist
-  maybe (return ()) removeFileIfExists captureFp
-
+  timestamp <- formatTime defaultTimeLocale (iso8601DateFormat (Just "%H-%M-%S")) <$> getCurrentTime
```

Comment:
Probably best to add yymmdd as well.

Reply:
[iso8601DateFormat](https://hackage.haskell.org/package/time-1.9.1/docs/Data-Time-Format.html#v:iso8601DateFormat) adds that for us